docs/source/index.rst

```diff
@@ -18,8 +18,8 @@ Contents
 
     models
     simulation
+    figures
 
 Search
```

Comment:
Sorry I mislead you here, I think it should be 
```
Search
=====
```
so search will be some kind of header.